packages/icons/scripts/icons.js

```diff
@@ -20,16 +17,41 @@ fetchImages({
 }).then(async svgs => {
   console.log("Building icons...");
 
-  const jsx = await Promise.all(svgs.map(svg => svgtojsx(svg.buffer)));
-  const data = svgs
-    .map((svg, index) => `export const ${pascalCase(svg.name)}Icon = () => ${jsx[index]}\n`)
+  const allIconExports = (
+    await Promise.all(
+      svgs.map(svg =>
+        transform(
+          svg.buffer.toString(),
+          {
+            icon: 20,
```

Comment:
Do we have any automated checks that ensure the icon view boxes are all the same and what we expect?

Reply:
No, but this is a great idea! I'll get this in a follow-up PR and add some better logs and error handling all around.